working-groups/green-reviews/README.md

```diff
@@ -0,0 +1,33 @@
+# Green Reviews Working Group
+
+The Green Reviews Working Group operates under CNCF Environmental Sustainability TAG.
+
+The Green Reviews Working Group helps CNCF projects assess and improve their cloud native sustainability footprint. By integrating environmental sustainability reviews into release cycles and empowering end users to evaluate project sustainability, the group promotes long-term viability and environmental responsibility within the CNCF ecosystem.
+
+See the working group charter here: <https://github.com/cncf/tag-env-sustainability/tree/main/working-groups/green-reviews/charter.md>
+
+## Motivation
+
+CNCF projects are becoming increasingly concerned and aware of the challenges related to environmental sustainability. Their objective is to integrate an environmental sustainability project review in each release cycle. Furthermore, they strive to empower end users to assess the sustainability of the cloud native projects within their systems. 
+
+The purpose of the Green Reviews Working Group is to help CNCF projects to review their cloud native sustainability footprint.
+
+To support CNCF projects, project maintainers and contributors, there is a need for a systematic approach to assess every project's sustainability footprint. Implementing a structured review process for this purpose would greatly benefit the CNCF ecosystem, establishing it as a best practice. Such a process would enable stakeholders to comprehensively evaluate and enhance the sustainability of CNCF projects, promoting their long-term viability and environmental responsibility. 
+
+Green Reviews Working Group empowers the CNCF Environmental Sustainability TAG to pursue its goals of promoting environmentally sustainable practices in the CNCF ecosystem.
+
+## Chairs
+
+- Niki Manoledaki, [@nikimanoledaki](http://github.com/nikimanoledaki)
+- Kristina Devochko, [@guidemetothemoon](http://github.com/guidemetothemoon)
+
+## Meetings and Contact
+
+The Green Reviews Working Group utilizes TAG Environmental Sustainability channels for communication.
+
+- Meeting time and frequency: TBA.
```

Comment:
Maybe you can propose here already a time e.g. every other Wednesday or so

Reply:
How about the following so that it alternates with the regular meeting? 
```suggestion
- Meeting time and frequency: Every 2nd & 4th Wednesday, 5pm GMT +2:00, 8AM Pacific, 11AM Eastern
```

Reply:
Thank you for the suggestion, Niki and Max, updated with suggested time and frequency.